api/src/main/java/org/openmrs/module/fgh/mpi/miscellaneous/AuthenticationType.java

```diff
@@ -0,0 +1,14 @@
+package org.openmrs.module.fgh.mpi.miscellaneous;
```

Comment:
Why do you need a new package named miscellaneous? Should be okay to use the existing package

Reply:
Which package do you suggest for this enum?

Reply:
Just don't introduce the new package, just put the classes in the same package like all existing classes

Reply:
Done